inst/bin/setenv.sh

```diff
@@ -1,7 +1,14 @@
 # Example:
 #export LD_LIBRARY_PATH=/usr/lib/oracle/11.2/client64/lib:$LD_LIBRARY_PATH
-LD_LIBRARY_PATH=/usr/lib/jvm/java-1.8.0/jre/lib/amd64:/usr/lib/jvm/java-1.8.0/jre/lib/amd64/server/
-PATH=/usr/pgsql-9.4/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ARCH=$(uname -m)
+
+if [ "$ARCH" = "x86_64" ]; then
+    LD_LIBRARY_PATH=/usr/lib/jvm/java-1.8.0/jre/lib/amd64:/usr/lib/jvm/java-1.8.0/jre/lib/amd64/server/:/R-4.4.0/lib
+elif [ "$ARCH" = "aarch64" ]; then
+    LD_LIBRARY_PATH=/usr/lib/jvm/java-1.8.0/jre/lib/aarch64:/usr/lib/jvm/java-1.8.0/jre/lib/aarch64/server/:/R-4.4.0/lib
+fi
+
+export LD_LIBRARY_PATH
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
I'm not particularly sure how this was working before because the R libs and java packages were also in a specific dir per arm and amd64 but with the upgrade I was getting errors in R loading java and stats packages and these changes fixed it.